### PR TITLE
feat: added StatusBar component to road-to-control page

### PR DESCRIPTION
**Pull request by johnlhuangP**

### CODE REVIEW

**Review by arielshif — COMMENTED** [review]
This looks awesome!

File: election-portal/app/road-to-control/modules/statusbar.module.css
Comment (arielshif) [inline election-portal/app/road-to-control/modules/statusbar.module.css:1]: Great stuff. Would love if all colors were referenced as variables from global styles.

Comment (akslaym) [reply election-portal/app/road-to-control/modules/statusbar.module.css:1]: Agreed. My only other comment would be to maybe see if you could round the top right corner of blue side and top left corner of red side kind of like in the Figma. This is an extreme detail and not needed. Looking great though. Thank you for your work John! I will merge as soon as you commit these changes.

**Review by arielshif — APPROVED** [review]

File: election-portal/app/road-to-control/modules/statusbar.module.css
Comment (akslaym) [reply election-portal/app/road-to-control/modules/statusbar.module.css:1]: Sounds good! I think we are good to go then. I will merge

**Pull request merged**